### PR TITLE
Update how-to-contribute.md

### PR DESCRIPTION
**Pull request by valoni01**
- [ ] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by valoni01** [comment]
I thought it would be nice to end the '!' to '.'  since it was the end of the sections.

File: _articles/how-to-contribute.md [hunk 1]
```diff
@@ -524,7 +524,7 @@ If you don't have time to work on the issue anymore (for example, if the convers
 
 ### 👎 Your contribution doesn't get accepted.
 
-Your contribution may or may not be accepted in the end. Hopefully you didn't put too much work into it already. If you're not sure why it wasn't accepted, it's perfectly reasonable to ask the maintainer for feedback and clarification. Ultimately, however, you'll need to respect that this is their decision. Don't argue or get hostile. You're always welcome to fork and work on your own version if you disagree!
+Your contribution may or may not be accepted in the end. Hopefully you didn't put too much work into it already. If you're not sure why it wasn't accepted, it's perfectly reasonable to ask the maintainer for feedback and clarification. Ultimately, however, you'll need to respect that this is their decision. Don't argue or get hostile. You're always welcome to fork and work on your own version if you disagree.
 
 ### 🎉 Your contribution gets accepted.
```
Comment (UttaM1990) [inline _articles/how-to-contribute.md:529]: #2498

**Review by UttaM1990 — APPROVED** [review]

**Comment by MaryJaneLove4** [comment]
Correct

**Review by UttaM1990 — APPROVED** [review]

**Review by MelnikovAG — APPROVED** [review]

**Review by MaryJaneLove4 — APPROVED** [review]
Ok

File: _articles/how-to-contribute.md (hunk 1 above)
Comment (MelnikovAG) [reply _articles/how-to-contribute.md:529]: Заболел тяжелой формой, COVID19, работать тяжело, приношу извинения, как станет лучше, сразу приступлю к работе - на 100 %

**Comment by MaryJaneLove4** [comment]
Am sorry. Am beginner. I can't do IT:(

Dne so 7. 8. 2021 22:20 uživatel ***@***.*** <
***@***.***> napsal:

**Pull request closed without merging**